### PR TITLE
Feature/release tag

### PR DESCRIPTION
**Pull request by carloscip**
Hability to build from release tag, and specify fcu majority flag via environment variable

### CODE REVIEW

File: dockerized/scripts/init.sh [hunk 1]
```diff
@@ -1,4 +1,11 @@
...
-executionbackup -p $PORT -n $NODES --listen-addr 0.0.0.0 --jwt-secret /scripts/.jwt --log-level $LOG_LEVEL
\ No newline at end of file
+executionbackup \
+-p $PORT \
+-n $NODES \
+--listen-addr 0.0.0.0 \
+--jwt-secret /scripts/.jwt \
+--log-level $LOG_LEVEL \
+--fcu-majority $FCU_MAJORITY \
+--node-timings
```
Comment (TennisBowling) [inline dockerized/scripts/init.sh]: We shouldn't enable node timings as it's decently verbose and was intended to be used to monitor slow performing nodes, or as a simple debugging solution

File: dockerized/default-env [hunk 2]
```diff
@@ -1,4 +1,6 @@
 NODES=<comma separated ndoes>
 PORT=<port to expose>
 LOG_LEVEL=<log level>
-JWT_SECRET=<jwt secret>
\ No newline at end of file
+JWT_SECRET=<jwt secret>
+RELEASE_TAG=<specify release tag if desired stable release build>
+FCU_MAJORITY=0.6
```
Comment (TennisBowling) [inline dockerized/default-env]: No need to specify the fcu majority flag here as it's 0.6 as it's set by default if not set.

**Review by TennisBowling — CHANGES_REQUESTED** [review]
Looks great!

File: dockerized/default-env (hunk 2 above)
Comment (carloscip) [reply dockerized/default-env]: thanks for the suggestions, fixed on a new commit

**Comment by TennisBowling** [comment]
thanks a lot!

**Pull request merged**